packages/dds/tree/src/test/simple-tree/schemaFactory.spec.ts

```diff
@@ -25,6 +25,8 @@ import {
 } from "../../simple-tree/schemaFactory.js";
 import { areSafelyAssignable, requireAssignableTo, requireTrue } from "../../util/index.js";
 import { TreeFactory } from "../../treeFactory.js";
+// eslint-disable-next-line import/no-internal-modules
```

Comment:
Is this eslint disable actually required? We use module-internal imports above.

Reply:
Yeah, it is...  One line above what GH is showing you there's a similar disablement for '../../simple-tree/schemaFactory.js'.

The question is, why do only some imports require them?  (I didn't investigate.)